app/models/server/models/Sessions.js

```diff
@@ -47,6 +47,7 @@ export const aggregates = {
 					month: '$month',
 					year: '$year',
 				},
+				mostImportantRole: { $first: '$mostImportantRole' },
```

Comment:
using `$first` it might not respect the rule os "most important role" right? I was thinking in a way to solve this and one solution I came up with was adding "scores" to roles and saving them on this collection along with the role name, this way we could use `$max` (or `$min`) instead of `$first`.. wdyt?

maybe the amount of times this can happen might not be as relevant for such complex change.

Reply:
@sampaiodiego since it's grouped by user it will be always the same value, will change only if the user changed his role during the day, and then we are considering the first record of the day, I don't see a problem, what do you think?

Reply:
> will change only if the user changed his role during the day

I was considering this exactly scenario.. but indeed I don't think it is a big problem not consider the "most important role" only on that day, for that user.